src/packages/@ncigdc/modern_components/CohortComparison/FacetTable.js

```diff
@@ -1,184 +1,191 @@
 import React from 'react';
 import { compose } from 'recompose';
 import { union, find, truncate } from 'lodash';
-import Table, { Tr, Td, Th } from '@ncigdc/uikit/Table';
-import { Row } from '@ncigdc/uikit/Flex';
 import ExploreLink from '@ncigdc/components/Links/ExploreLink';
 import BarChart from '@ncigdc/components/Charts/TwoBarChart';
 import { withTheme } from '@ncigdc/theme';
 import Pvalue from '@ncigdc/modern_components/Pvalue';
+import { visualizingButton } from '@ncigdc/theme/mixins';
+import Button from '@ncigdc/uikit/Button';
+import toTsvString from '@ncigdc/utils/toTsvString';
+import saveFile from '@ncigdc/utils/filesaver';
+import EntityPageHorizontalTable from '@ncigdc/components/EntityPageHorizontalTable';
 
 export default compose(
   withTheme,
 )(
   ({
     theme,
-    mapping,
     field,
     data1,
     data2,
     result1,
     result2,
-    Set1,
-    Set2,
     set1,
     set2,
     Alias,
     palette,
-    buckets1 = data1[field].buckets.filter(x => x.key !== '_missing'),
-    buckets2 = data2[field].buckets.filter(x => x.key !== '_missing'),
-    data = union(buckets1.map(b => b.key), buckets2.map(b => b.key)),
-  }) =>
-    <div>
-      <Row>
-        <h2>{mapping[field]}</h2>
-        {/* <Row style={{ marginLeft: 'auto', alignItems: 'center' }}>
-        <div>{Set1}</div>
-        <div style={{ margin: '0 10px' }}>{Set2}</div>
-      </Row> */}
-      </Row>
-      <BarChart
-        data1={data.map(k => {
-          const bucket = find(buckets1, b => b.key === k) || {};
+    heading,
+  }) => {
+    const buckets1 = data1[field].buckets.filter(x => x.key !== '_missing');
+    const buckets2 = data2[field].buckets.filter(x => x.key !== '_missing');
+    const tableData = union(
+      buckets1.map(b => b.key),
+      buckets2.map(b => b.key),
+    ).map(k => {
+      const casesS1 = (find(buckets1, b => b.key === k) || {}).doc_count || 0;
+      const casesS2 = (find(buckets2, b => b.key === k) || {}).doc_count || 0;
 
-          return {
-            label: truncate(k, { length: 15 }),
-            value: bucket.doc_count || 0,
-          };
-        })}
-        data2={data.map(k => {
-          const bucket = find(buckets2, b => b.key === k) || {};
+      return {
+        value: k,
```

Comment:
should this be key or label or something? value might not be specific enough

Reply:
I'm not sure for the audience. Key seemed to dev centric. @phuongmy do you have an opinion?

Reply:
maybe I'm confused as to how that field is used - but why would @phuongmy have an opinion on our variable names?

do we display those as strings on the UI? Like if you changed it to

```
return {
  beep: k
```

somewhere in the UI it would say `beep`?

Reply:
oh does it get rendered in the TSV?

Reply:
if so - maybe `term`?

Reply:
yeah, it gets rendered in the TSV. term works for me :D